tests/longevity-2mv-backpressure-4d.yaml

```diff
@@ -0,0 +1,77 @@
+test_duration: 5760
+prepare_write_cmd: "cassandra-stress user profile=/tmp/c-s_profile_2mv_2queries.yaml ops'(insert=1)' cl=QUORUM n=250000000 -port jmx=6868 -mode cql3 native -rate threads=200"
+stress_cmd: ["cassandra-stress user profile=/tmp/c-s_profile_2mv_2queries.yaml ops'(insert=1)' cl=QUORUM duration=5760m -port jmx=6868 -mode cql3 native -pop seq=250000000..500000000 -rate threads=200",
+            "cassandra-stress user profile=/tmp/c-s_profile_2mv_2queries.yaml ops'(mv_p_read1=1,mv_p_read2=1)' cl=QUORUM duration=5760m -port jmx=6868 -mode cql3 native -pop seq=1..250000000 -rate threads=10"]
+n_db_nodes: 4
+n_loaders: 4
+n_monitor_nodes: 1
+user_prefix: 'longevity-2mv-backpressure-4d-VERSION'
+nemesis_class_name: 'ChaosMonkey'
+nemesis_interval: 30
+nemesis_during_prepare: 'false'
+failure_post_behavior: keep
+space_node_threshold: 644245094
```

Comment:
Is it needed? will it start nemesis?

Reply:
Yes, after the https://github.com/scylladb/scylla-cluster-tests/pull/769 will be merged